mailbox/tools/quota-recompute/src/main/java/org/apache/james/mailbox/quota/task/RecomputeCurrentQuotasService.java

```diff
@@ -141,7 +141,7 @@ public RecomputeCurrentQuotasService(UsersRepository usersRepository,
     public Mono<Task.Result> recomputeCurrentQuotas(Context context) {
         try {
             return Iterators.toFlux(usersRepository.list())
-                .flatMap(username -> recomputeUserCurrentQuotas(context, username))
+                .concatMap(username -> recomputeUserCurrentQuotas(context, username))
```

Comment:
the commit message should explain a bit the "why" because as-is, it only repeat what I can read in the code

Reply:
I did a rebase to reword that commit and add it a message.